boltdesignsystem/bolt
# DS-138 Replace Button/Link in PL and Jest tests

## Jira

[https://pegadigitalit.atlassian.net/browse/DS-138](https://pegadigitalit.atlassian.net/browse/DS-138)

## Summary

Fully remove button and link from the components directory

## Details

We're deprecating and removing the button component. This replaces the remaining instances of the button and link components internally.

## How to test

- Review code and demos for any potential breaking changes.
- Confirm there are no other instances of button and link in Bolt that should be replaced (I have replaced all the ones that I thought should be, but there were some hidden files and tests that I was unsure about).
- block-list is broken, but no one uses it and we're removing it.
- Search the whole repo for something other than `button.twig` and `<bolt-button` and `link.twig` and `>bolt-link`, this is to confirm (and double check) that all these components have been tracked. These are the search terms I used, so they may be helpful for bullet 2.

## Release notes

### Visual changes

Button text size is expected to change.
